src/routes/products/idsSearcher.js

```diff
@@ -0,0 +1,45 @@
+const products = require("../../db/products/all-products.json");
+const qs = require("querystring");
+
+const idsSearcher = (request, response) => {
+  let productsArraySearchResult = [];
+  let productsToShow;
+
+  const idsObj = qs.parse(request.url);
+  const idsStringFromUrl = Object.values(idsObj)[0];
+  const idsFirstIndex = idsStringFromUrl.indexOf("'");
+  const idsLastIndex = idsStringFromUrl.lastIndexOf("'");
+  const idsValue = idsStringFromUrl
+    .slice(idsFirstIndex + 1, idsLastIndex)
+    .split(",");
+  console.table(idsValue);
+
+  idsValue.map(item => {
```

Comment:
Let's imagine situation, when you don't have product with the last id passed `?ids='1','2','3','wrong_id'`. Then your client receive `no products` status. It's better to check results after iteration through ids